presentation/common/src/presentation-common/PresentationRpcInterface.ts

```diff
@@ -197,38 +197,45 @@ export class PresentationRpcInterface extends RpcInterface {
     NOTE: Please consult the README in core/common/src/rpc for the semantic versioning rules.
   ===========================================================================================*/
 
+  /** @deprecated in 4.10. Use [PresentationManager]($presentation-backend) instead of calling the RPC interface directly. */
```

Comment:
`master` is now pointing to `5.0` - are you going to rebase this branch to `release/4.10.x`?